Common/SM_datatypes.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "PM_datatypes.hpp"
+#include "AM_datatypes.hpp"
+
+struct SM_TM_Commands {
+    GroundStationData gspcData;
+    JetsonMovementRequest jetson_movement_req;
+    GroundStationPIDSetResponse gspc_pid_set_resp;
+};
+
+struct TM_SM_Commands {
+    GroundStationDisarm ground_station_disarm;
+    WaypointsCommand waypoint_commands;
+    JetsonToZpMovementCommand jetson_commands; 
+    LandingInitiationCommand landing_initiation; 
+    GroundStationPIDSetCommand gspc_pid_set_command;
+};
+
+struct GroundStationDisarm {
+    bool arm;
+};
+
+struct GroundStationData {
+    uint8_t motor_outputs[12];
+    LosSFData sf_data;
+    uint8_t battery_voltages[13];
+    uint8_t controller_values[16];
+};
```

Comment:
@mikashaw could you get mihir to take a look at this again :D .

But I think you might also want to include something like queued waypoints or smth so that the ground knows what the drone does.